resolwe_bio/processes/icount/annotation.yml

```diff
@@ -0,0 +1,38 @@
+- slug: icount-annotation
+  name: iCount annotation
+  description: >
+    Download annotation for given release/species from ENSEMBL.
+  category: import
+  version: 1.0.0
+  type: data:annotation:gtf
+  requirements:
+    expression-engine: jinja
+    executor:
+      docker:
+        image: "resolwe/bio-linux8-resolwe-preview:icount"
+  data_name: 'ENSEMBL annotation ({{ species ~ "." ~ release | default("?")}})'
+  persistence: RAW
+  input:
+  - name: species
+    label: Species latin name.
+    type: basic:string
+  - name: release
```

Comment:
There are releases from 59 - 84, I would't bother including them all. Or should I?